File: solutions/python/resistor-color-duo/1/resistor_color_duo.py
""" Module doc string """
res_val = {"black": 0, "brown": 1, "red": 2,
           "orange": 3, "yellow": 4, "green": 5,
           "blue": 6, "violet": 7, "grey": 8, "white": 9}

def value(colors):
    """ function doc string"""
    return res_val[colors[0]] * 10 + res_val[colors[1]]

File: solutions/csharp/resistor-color-duo/1/ResistorColorDuo.cs
using System;
using System.Collections.Generic;

public static class ResistorColorDuo
{
    static Dictionary<string,int> cv = new Dictionary<string, int>() {{"black", 0},{"brown", 1}, {"red", 2}, {"orange", 3},
        {"yellow", 4}, {"green", 5}, {"blue", 6},
        {"violet", 7}, {"grey", 8}, {"white", 9}};
    public static int Value(string[] colors)
    {
        return 10 * cv[colors[0]] + cv[colors[1]];
    }
}
